python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -580,6 +585,14 @@ def _run_iteration_impl(
             elif is_under_min_interval(auto_materialize_state, sensor):
                 continue
 
+            self.instrument_elapsed(
+                sensor,
+                get_elapsed(auto_materialize_state, sensor)
+                if auto_materialize_state and sensor
```

Comment:
its confusing but AMP can have a 'global' auto_materialize_state even if there is no RemoteSensor object so i think you can still compute this even if there is no sensor